docs/tests.md

```diff
@@ -1,33 +1,84 @@
-- **subpath**: the `subpath` corresponding to this `purl`.
-- **is_invalid**: a boolean flag set to true if the test should report an
-  error
-
-To test `purl` parsing and building, a tool can use this test suite and for
-every listed test object, run these tests:
-
-- parsing the test canonical `purl` then re-building a `purl` from these
-  parsed components should return the test canonical `purl`
-
-- parsing the test `purl` should return the components parsed from the test
-  canonical `purl`
-
-- parsing the test `purl` then re-building a `purl` from these parsed
-  components should return the test canonical `purl`
-
-- building a `purl` from the test components should return the test canonical
-  `purl`
+The Package-URL (PURL) specification provides a JSON Schema and test
+files to support language-neutral testing of PURL implementations. The 
+objectives for the PURL schema and test files are to enable tools to conform
+to the PURL specification for tool functions such as:
+- build a canonical PURL string from a set of PURL component data
+- parse a canonical PURL string into a set of PURL components
+- parse a PURL string input and rebuild it as a canonical PURL string
+
+The current  JSON schema is available at: [`purl-spec/schemas/purl-test.schema-0.1.json`](https://github.com/package-url/purl-spec/blob/main/schemas/purl-type-definition.schema-1.0.json).
+
+The test files are available at:
+- [`purl-spec/tests/spec/`](https://github.com/package-url/purl-spec/tree/main/tests/spec): 
+This folder contains JSON test files that are not for a specific PURL type.
+  - [`specification-test.json`](https://github.com/package-url/purl-spec/blob/main/tests/spec/specification-test.json) - This file contains an array of test objects
+  that primarily cover testing the validity of individual PURL components,
+  separators between PURL components, and complete PURL strings.
+  - There is a proposal to add separate test files in this folder for each 
+  PURL component.
+- [`purl-spec/tests/types/`](https://github.com/package-url/purl-spec/tree/main/tests/types): This folder contains one JSON test file for each registered PURL type. These 
+tests should be focused on test cases that are specific to a PURL type, such 
+as those for namespace or qualifiers.
+
+Two key properties in the PURL test JSON schema are:
+- Test groups
+- Test types
+
+### Test groups
+
+There are two PURL test groups:
+- **base**: Test group for base conformance tests. Base tests are pass/fail.
+- **advanced**: Test group for advanced tests. Advanced tests are more 
+permissive than base tests. They may correct some errors.
+
+### Test types
+
+There are three PURL test types:
+- **build**: A test to build a canonical PURL output string from an input of 
+decoded PURL components. See also [`/docs/how-build.md`](https://github.com/package-url/purl-spec/blob/main/docs/how-to-build.md).
+- **parse**: A test to parse decoded components from a canonical PURL 
```

Comment:
Consider this instead:  parse: A test to parse a canonical PURL input string into a set of decoded components.  (I'll make a similar suggested change in the schema....)